ti/instrumentation/java/helper.go

```diff
@@ -67,6 +70,72 @@ func getFiles(path string) ([]string, error) {
 	return matches, err
 }
 
+// GetJavaTests returns list of RunnableTests in the workspace with java extension.
+// In case of errors, return empty list
+func GetJavaTests(workspace string) []ti.RunnableTest {
+	tests := make([]ti.RunnableTest, 0)
+	files, _ := getFiles(fmt.Sprintf("%s/**/*.java", workspace))
+	for _, path := range files {
+		if path == "" {
+			continue
+		}
+		node, _ := ParseJavaNode(path)
+		if node.Type != NodeType_TEST {
+			continue
+		}
+		test := ti.RunnableTest{
+			Pkg:   node.Pkg,
+			Class: node.Class,
+		}
+		tests = append(tests, test)
+	}
+	return tests
+}
+
+// GetScalaTests returns list of RunnableTests in the workspace with scala extension.
+// In case of errors, return empty list
+func GetScalaTests(workspace string) []ti.RunnableTest {
+	tests := make([]ti.RunnableTest, 0)
+	files, _ := getFiles(fmt.Sprintf("%s/**/*.scala", workspace))
+	for _, path := range files {
+		if path == "" {
+			continue
+		}
+		node, _ := ParseJavaNode(path)
```

Comment:
Should we split ParseJavaNodeinto 3 different functions one for each language?

Reply:
We treat scala/kotlin the same as java in the backed it. Separate them is fine if you think the function gets too long

Reply:
Let's keep it consistent with addon